app/app.py

```diff
@@ -18,42 +19,51 @@ def create_app() -> Flask:
 
     server = Flask(__name__)
 
-    database_service = DatabaseService()
-    figure_service = FigureService(database_service)
+    server.database_service = DatabaseService()
+    server.figure_service = FigureService(server.database_service)
+
+    configure_routes(server)
 
     logger.info("Populating stub data...")
-    database_service.create_indicators_table()
-    database_service.clean_indicators_table()
-    database_service.add_data()
-
-    app = Dash(__name__, server=server)
-
-    app.layout = html.Div(
-        children=[
-            dcc.Graph(
-                figure=figure_service.get_number_of_repositories_with_standards_label_dashboard(),
-                style={"width": "33%", "height": "500px", "display": "inline-block"},
-            ),
-            dcc.Graph(
-                figure=figure_service.get_number_of_repositories_archived_by_automation(),
-                style={"width": "33%", "height": "500px", "display": "inline-block"},
-            ),
-            dcc.Graph(
-                figure=figure_service.get_sentry_transactions_used(),
-                style={"width": "33%", "height": "500px", "display": "inline-block"},
-            ),
-            dcc.Graph(
-                figure=figure_service.get_support_stats(),
-                style={"width": "100%", "height": "500px", "display": "inline-block"},
-            ),
-            dcc.Graph(
-                figure=figure_service.get_github_actions_quota_usage(),
-                style={"width": "100%", "height": "500px", "display": "inline-block"},
-            ),
-        ],
-        style={"padding": "0px", "margin": "0px", "background-color": "black"},
-    )
+    server.database_service.create_indicators_table()
+    server.database_service.clean_stubbed_indicators_table()
+    server.database_service.add_stubbed_indicators()
+
+    app = Dash(__name__, server=server, url_base_pathname="/dashboard/")
+    app.title = "⚙️ OE - KPI Dashboard"
+    app.layout = create_dashboard(server.figure_service)
```

Comment:
The layout needs to be a function Callable to easily enable reloading on page refresh ♻️ https://dash.plotly.com/live-updates